src/services/test-result.ts

```diff
@@ -0,0 +1,77 @@
+import { Callback } from 'aws-lambda';
+import createResponse from '../utils/create-response';
+import { DynamoDB, AWSError } from 'aws-sdk';
+import * as UUID from 'uuid'
+import { ITestResult, IResponse } from '../interfaces/interface';
+
+export default class TestResultService {
+
+	constructor(private db: DynamoDB.DocumentClient, private tableName: string) { }
+	
+	create(requestBody: any, callback: Callback) {
+		const testResultData: ITestResult = this.extractTestResult(requestBody);
```

Comment:
Do we want to link JSON to the database like this? 
Should we have objects in between?

Reply:
I'm afraid that I didn't get you.
the extractTestResult will extract just properties that I need